app/filtering.js

```diff
@@ -412,23 +412,23 @@ function registerForDownloadListener (session) {
   })
 }
 
-function registerSession (partition, fn) {
```

Comment:
this used to call `registeredSessions[partition] = ses` once for every function in `fns`. pretty sure that was not needed?